src/lib/constants.ts

```diff
@@ -24,13 +28,14 @@ const constants = {
 		checkString(process.env.HOST_OS_VERSION_PATH) ||
 		withRootMount('/etc/os-release'),
 	// Data paths
+	dataMountPoint,
 	databasePath:
 		checkString(process.env.DATABASE_PATH) || '/data/database.sqlite',
 	appsJsonPath:
 		process.env.APPS_JSON_PATH || withRootMount('/mnt/data/apps.json'),
```

Comment:
I don't think we set `APPS_JSON_PATH` anywhere, so probably this should default to `/mnt/data/apps.json`

Reply:
Removed the env var [here](https://github.com/balena-os/balena-supervisor/pull/2120/files#diff-8c943ccfb5928fff63708e99eaafa93f8845c9405ed8e4b32afed23e5003025aR32)